src/flow_solver/flow_solver_cases/periodic_cube_flow.cpp

```diff
@@ -50,26 +54,11 @@ std::shared_ptr<Triangulation> PeriodicCubeFlow<dim,nstate>::generate_grid() con
         if constexpr(dim == 3)
         {
 
-            std::cout << "Using Input Mesh -- GMSH_READER." << std::endl;
+            pcout << "Using Input Mesh -- GMSH_READER." << std::endl;
 
             const std::string mesh_filename =
                     this->all_param.flow_solver_param.input_mesh_filename + std::string(".msh");
-            std::shared_ptr <HighOrderGrid<dim, double>> Cube_mesh = read_gmsh<dim, dim>(mesh_filename);
-
-            /**
-             * Adding Periodic BC for GMSH (Note that the BC number are dependent on the mesh)
-             * Currently accepted convention is
-             *  -> 2001 & 2002 for periodic faces on the x-axis
-             *  -> 2003 & 2004 for periodic faces on the y-axis
-             *  -> 2005 & 2006 for periodic faces on the z-axis
-             */
-            std::vector <dealii::GridTools::PeriodicFacePair<typename dealii::Triangulation<dim>::cell_iterator>> matched_pairs;
-            dealii::GridTools::collect_periodic_faces(*Cube_mesh->triangulation, 2001, 2002, 0, matched_pairs);
-            dealii::GridTools::collect_periodic_faces(*Cube_mesh->triangulation, 2003, 2004, 1, matched_pairs);
-            dealii::GridTools::collect_periodic_faces(*Cube_mesh->triangulation, 2005, 2006, 2, matched_pairs);
-            Cube_mesh->triangulation->add_periodicity(matched_pairs);
-
-            std::cout << "Finish adding periodic boundary conditions." << std::endl;
+            std::shared_ptr <HighOrderGrid<dim, double>> Cube_mesh = read_gmsh<dim, dim>(mesh_filename, this->all_param.flow_solver_param.use_periodic_BC_in_x, this->all_param.flow_solver_param.use_periodic_BC_in_y, this->all_param.flow_solver_param.use_periodic_BC_in_z, this->all_param.flow_solver_param.x_periodic_id_face_1, this->all_param.flow_solver_param.x_periodic_id_face_2, this->all_param.flow_solver_param.y_periodic_id_face_1, this->all_param.flow_solver_param.y_periodic_id_face_2, this->all_param.flow_solver_param.z_periodic_id_face_1, this->all_param.flow_solver_param.z_periodic_id_face_2);
```

Comment:
only use capital letters for namespaces and class names

Reply:
Fixed this in the recent commit.